cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -188,6 +188,13 @@ public void mockStateEndpoint(CruiseControlUserTaskStatus taskStatus, boolean fe
             return;
         }
 
+        if (taskStatus == null) {
+            server
+                    .when(requestMatcher(authHeader, CruiseControlEndpoints.STATE, "true", "false"))
+                    .respond(buildStateResponse("CC-State.json", "cruise-control-state", 200));
```

Comment:
This is really weird formatting (at least I think minus one \t) would be better)

Reply:
I agree, but the spacing/tabbing is consistent with the existing lines of the class. Would you be ok if we refactored the spacing/tabs of this class in a separate PR? I want the focus of the reviewers to be on the logic changes and not on formatting changes. WDYT?

Reply:
You might not be allowed to change the indentatiom. The main question here is whether you really need to have a new line there.

```suggestion
            server.when(requestMatcher(authHeader, CruiseControlEndpoints.STATE, "true", "false"))
                    .respond(buildStateResponse("CC-State.json", "cruise-control-state", 200));
```

Reply:
Removed new lines.